app/models/manageiq/providers/nuage/network_manager/cloud_subnet.rb

```diff
@@ -1,2 +1,60 @@
 class ManageIQ::Providers::Nuage::NetworkManager::CloudSubnet < ::CloudSubnet
+  supports :delete
+  supports :create
+
+  def delete_cloud_subnet_queue(userid)
+    task_opts = {
+      :action => "deleting Cloud Subnet for user #{userid}",
+      :userid => userid
+    }
+    queue_opts = {
+      :class_name  => self.class.name,
+      :method_name => 'raw_delete_cloud_subnet',
```

Comment:
We don't usually (ever?) queue the `raw_*` methods directly, we would queue `delete_cloud_subnet` which would then call `raw_delete_cloud_subnet` directly

Reply:
Oh I see, this is how openstack does it.  I'm okay keeping it consistent with how they define it then.

Reply:
(I would lean to fix it here, then in OpenStack)

Reply:
Yeah, even for OpenStack it's not the same everywhere (delete_cloud_subnet_queue invokes raw directly, while create_cloud_subnet_queue does the additional hop as you suggest). Thanks for guideline, I'll do as you suggest.